### PR TITLE
incorporate plone.synchronize

### PR DESCRIPTION
**Pull request by jensens**
`plone.synchronize` is one function plus a test and not worth an own package. it is used only in `plone.dexterity.schema`

### CODE REVIEW

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

File: news/157.bugfix [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Incorporate `plone.synchronize` its one and only simple `synchronized` function into `plone.dexterity`, which were the only consumer.
+Also moves the test. Removes a dependency and a packgae to look after.
```
Comment (ale-rt) [inline news/157.bugfix:2]: packgae -> package

Really minor

File: news/157.bugfix [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Incorporate `plone.synchronize` its one and only simple `synchronized` function into `plone.dexterity`, which were the only consumer.
```
Comment (ale-rt) [inline news/157.bugfix:1]: I think you want to double the "`" to have a good rst

**Review by ale-rt — APPROVED** [review]
LGTM, minor fixes would be nice to have in the towncrier entry.
I do not see the urge to get rid of plone.synchronize (I think it will never get updated) but it seems sane.

**Comment by ale-rt** [comment]
Thanks @jensens for tolerating my picky reviews ;)

**Comment by jensens** [comment]
@ale-rt they are perfect, that is what review are for.

**Pull request merged**